synadm/api.py

```diff
@@ -411,10 +411,22 @@ def user_list(self, _from, _limit, _guests, _deactivated,
 
     def user_list_paginate(self, _limit, _guests, _deactivated,
                            _name, _user_id, _from="0"):
+        # documentation is mostly duplicated from user_list...
         """Yields API responses for all of the pagination.
 
         Args:
-            _from (string): Opaque string
+            _limit (int): maximum number of users returned, used for pagination
+            _guests (bool): enable/disable fetching of guest users
+            _deactivated (bool): enable/disable fetching of deactivated users
+            _name (string): user name localpart to search for, see Synapse
+                admin API docs for details
+            _user_id (string): fully qualified Matrix user ID to search for
+            _from (string): offsets user list by this number, used for
+                pagination
+
+        Yields:
+            dict: The admin API response for listing accounts.
+                https://matrix-org.github.io/synapse/latest/admin_api/user_admin_api.html#list-accounts
```

Comment:
I kind of like the idea of documenting by linking to the actual API docs. I'd like to continue discussing that idea for  throughout synadm in#synadm. I did something like that with a general statement on the very top of  api.py IIRC but let's discuss if we would like to explicitely state a link in every API method.